src/ReleaseHistory.md

```diff
@@ -1,5 +1,10 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fixed `merge` command producing runs per rule to the expected runs per tool. [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
```

Comment:
>Fixed [](http://example.com/codeflow?start=10&length=5)

Use this:

Fix `merge` command failure to produce runs by tool and version when passed `merge-runs`.